### DISCUSSION TITLE
Working with Google Gemini and ColdFusion

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#13] [Comments]
# Working with Google Gemini and ColdFusion

Father, husband, developer relations and web standards expert, and cat demo builder.

https://www.raymondcamden.com/2024/01/23/working-with-google-gemini-and-coldfusion

**Comment by VisationLtd** [comment]
Nice to see this article Ray. I'm a 25yr old CF vet in the UK but have yet to dip my toe in the interesting world of AI.. I think you have maybe just inspired me. Not for the first time! best wishes, Jonathan
